tests/page-objects/users/users.wdio.page.js

```diff
@@ -0,0 +1,36 @@
+const utils = require('../../utils');
+const commonElements = require('../common/common.wdio.page');
+
+const getAddUserButton = () => $('#add-user');
+
+module.exports = {
+  addUser: async (username, password) => {
```

Comment:
This looks strange. Is this used anywhere? 
Why are we trimming a string that we don't user in the end? 

Same goes for the other methods? They also look identical?

Reply:
removed this file and used existing one